packages/@webex/plugin-meetings/src/breakouts/breakout.ts

```diff
@@ -56,14 +58,38 @@ const Breakout = WebexPlugin.extend({
    * @returns {Promise}
    */
   join() {
-    return this.request({
+    const breakoutMoveId = uuid.v4();
+    const activeMeetingId = this.collection.parent.meeting.deref().id;
```

Comment:
its a hacky way to get the meeting id here cant we save the meeting id when the breakout session gets created